.github/templates/bazel_ci_workflow.yml.j2

```diff
@@ -60,6 +60,7 @@ name: Bazel Linux CI (!{{ build_environment }})
             -e CUSTOM_TEST_ARTIFACT_BUILD_DIR \
             -e SKIP_SCCACHE_INITIALIZATION=1 \
             -e TORCH_CUDA_ARCH_LIST \
+            -e http_proxy="!{{squid_proxy}}" -e https_proxy="!{{squid_proxy}}" -e no_proxy="!{{squid_no_proxy}}" \
```

Comment:
These (`squid_proxy`, etc.) look like they're not defined for this workflow? Since they're duplicated across templates anyways it makes sense to pull them out into `generate_ci_workflows.py` so they're more consolidated

Reply:
this bazel workflow is an extension of linux workflow, but with combined `build + test` jobs into one, so many environment setup are defined in linux jinja2.